README.md

```diff
@@ -8,7 +8,8 @@ Portfolio Mobile Version
 This project is to set up repository locally and on the github. Set up Linters configuration the project.
 The project is in a number of sections wich inclusde the headers, the name and title, the introduction and my abilities sections. This is my Portfolio which shows what I have done.
 I have added the resent project section 
-I have coverted the mobile version into a desk top version making it responsive(using a responsive design)
+I have coverted the mobile version into a desk top version making it responsive(using a responsive design).
+I have added a menu for the mobile version using event listeners.
```

Comment:
- [ ] You did a nice job documenting your code! 📜 However, we are now required to update the README file with the new [Microverse Template](https://github.com/microverseinc/readme-template). Please, don't forget to customize it with the relevant sections for this project as well as to include the logo. This brand-new template will give a nicer look to your repo! 🤓 🛠️